casanovo/casanovo.py

```diff
@@ -259,21 +289,11 @@ def setup_logging(
 
     Parameters
     ----------
-    output : Optional[str]
-        The provided output file name.
+    log_file_path: Path
+        The log file path.
     verbosity : str
         The logging level to use in the console.
-
-    Return
-    ------
-    output : Path
-        The output file path.
     """
-    if output is None:
-        output = f"casanovo_{datetime.datetime.now().strftime('%Y%m%d%H%M%S')}"
-
-    output = Path(output).expanduser().resolve()
```

Comment:
**question:** The `expanduser` and `resolve` are not reflected in the new functionality at the bottom of this file. Is this still being done? Explicitly expanding the user was necessary to ensure cross-platform functionality (e.g. paths with `~` can be resolved natively in Unix but not in Windows, which is why we explicitly did it ourselves). Please verify that this functionality is retained.